pkg/local_object_storage/metabase/version.go

```diff
@@ -71,3 +80,26 @@ func getVersion(tx *bbolt.Tx) uint64 {
 
 	return 0
 }
+
+var migrateFrom = map[uint64]func(*DB, *bbolt.Tx) error{
+	2: migrateFrom2Version,
+}
+
+func migrateFrom2Version(db *DB, tx *bbolt.Tx) error {
+	tsExpiration := db.epochState.CurrentEpoch() + objectconfig.DefaultTombstoneLifetime
+	bkt := tx.Bucket(graveyardBucketName)
```

Comment:
again can be nil

Reply:
all the code added in this PR works on either explicitly inited database or on the database with a known version (that has also been inited sometime ago). in this case added a check since this code works on a database that _may_ be corrupted by some different code but a user can do nothing with it and it is not a situation that can be handled somehow

Reply:
> How exactly node behaves on metabase panic currently?

this is done in the main routine, it stops